im-sdk-cli/src/main/java/com/easemob/im/cli/cmd/CreateCmd.java

```diff
@@ -217,7 +218,7 @@ public void message(@Parameters(description = "message sender") String sender,
         String roomId;
 
         @Option(names = "--super", description = "add a super admin, who is the only person that can create a room")
-        String superAdminUsername;
+        boolean superAdmin;
```

Comment:
Maybe we can rephrase it to:
'promote the specified user to super admin, super admin is some user who can create room'
And, we can use promote/demote when else where when describing roles above plain user.